packages/core/src/components/dropdown/dropdown.tsx

```diff
@@ -36,7 +37,11 @@ import { AlignedPlacement } from './placement';
 export type DropdownTriggerEvent = 'click' | 'hover' | 'focus';
 
 type DisposeDropdown = () => void;
-const dropdownDisposer = new Map<string, DisposeDropdown>();
+type DropdownDisposerEntry = {
+  element: HTMLStencilElement;
```

Comment:
Why do you use `HTMLStencilElement` instead of `HTMLIxDropdownElement`?

Reply:
https://github.com/siemens/ix/pull/789/commits/bb4c8db87e3f29f3723e0858c979e683fc44e62d